test/e2e_tests/dir_test.go

```diff
@@ -521,6 +526,70 @@ func (suite *dirTestSuite) TestGitStash() {
 	}
 }
 
+func (suite *dirTestSuite) TestReadDirLink() {
```

Comment:
This test is failing on the Windows CI/CD. This test needs to be skipped on Windows because the os.Symlink call requires admin rights.

Reply:
Ok!